doc/md/writing-motoko/intercanister-calls.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 11
+---
+
+# Inter-canister calls
+
+## Overview
+
+One of the most important features of ICP for developers is the ability to call functions in one canister from another canister. This capability to make calls between canisters, also sometimes referred to as **inter-canister calls**, enables you to reuse and share functionality in multiple dapps.
+
+For example, you might want to create a dapp for professional networking, organizing community events, or hosting fundraising activities. Each of these dapps might have a social component that enables users to identify social relationships based on some criteria or shared interest, such as friends and family or current and former colleagues.
+
+To address this social component, you might want to create a single canister for storing user relationships then write your professional networking, community organizing, or fundraising application to import and call functions that are defined in the canister for social connections. You could then build additional applications to use the social connections canister or extend the features provided by the social connections canister to make it useful to an even broader community of other developers.
+
+This example will showcase a simple way to configure inter-canister calls that can be used as the foundation for more elaborate projects and use-cases such as those mentioned above.
+
+## Example
```

Comment:
This example could be better. Maybe add an issue for that?

Reply:
Created an issue: https://github.com/dfinity/motoko/issues/4516